### PR TITLE
Fix realtime-client communication bugs

### LINKED ISSUE #294: Wrong identification of multiple clients
In some cases, realtime thinks that a client exists when there isn't.

Some scenarios:
(1)
- Realtime becomes live
- Client is loaded, connects to realtime
- Backend becomes live, client requests work from realtime
(2)
- Realtime, backend, client are live and working
- Realtime and backend die, client remains loaded
- Backend becomes live
- Realtime becomes live, client requests work

In both scenarios the realtime is stuck in a loop where the message "debug: There is an other victimId <-> client.id match. Make client idle" is printed.

### PR DESCRIPTION
**Pull request by dimkarakostas**
Fix #294

### CODE REVIEW

File: realtime/index.js [hunk 1]
```diff
@@ -160,9 +165,10 @@ socket.on('connection', (client) => {
     client.on('disconnect', () => {
         winston.info('Client ' + client.id + ' disconnected');
 
+        // If this client was active, empty the spot of its victim
         for (let i in victims) {
             if (victims[i] == client.id) {
-                victims[i] = null;
+                delete victims[i];
```
Comment (esarafianou) [inline realtime/index.js]: is there a delete command in python? Do you mean `del`?

File: realtime/index.js [hunk 2]
```diff
@@ -160,9 +165,10 @@ socket.on('connection', (client) => {
     client.on('disconnect', () => {
         winston.info('Client ' + client.id + ' disconnected');
 
+        // If this client was active, empty the spot of its victim
```
Comment (esarafianou) [inline realtime/index.js]: Update comment

File: realtime/index.js (hunk 1 above)
Comment (dimkarakostas) [inline realtime/index.js]: This is Javascript :P

Comment (esarafianou) [inline realtime/index.js]: Sure. My bad

**Pull request merged**